packages/viewer/cypress/e2e/space.spec.ts

```diff
@@ -0,0 +1,21 @@
+import { deduce, setup } from '../support'
+
+beforeEach(setup)
+
+it('links to properties', () => {
+  cy.visit('spaces/S000004/properties')
+  deduce()
```

Comment:
Because we are increasingly adding these "wait for deduction" blocks, we want to make sure that deduction run stays fast (and – importantly – within the timeout).